src/code/z_player_lib.c

```diff
@@ -1479,6 +1479,7 @@ void Player_PostLimbDrawGameplay(PlayState* play, s32 limbIndex, Gfx** dList, Ve
 
             CLOSE_DISPS(play->state.gfxCtx, "../z_player_lib.c", 2656);
         } else if ((this->actor.scale.y >= 0.0f) && (this->meleeWeaponState != 0)) {
+            // This check is why bottles do not draw when you have ISG: it does this instad of the next case.
```

Comment:
this feels really niche to be commenting on. its not a bug in itself, just a result of something weird after youve already broken the game. just seems like a slippery slope of what things are worth commenting on as a whole

Reply:
to try and articulate my point better: this code is under the correct assumption that you will not be swinging your sword at the same time as using a bottle. assuming of course that you didnt activate an unrelated bug to turn that into reality.

if we start commenting on the side effects from the results of all bugs, even on those that are valid code otherwise, there would be a lot of things to say (to the point of it not being worth it/relevant to comment on, in my opinion)

Reply:
Yeah basically OoT is too interesting to fit every comment that can be made in this codebase